### PR TITLE
Incident alerts

### PR DESCRIPTION
**Pull request by soullivaneuh**
Continue of #142

### CODE REVIEW

File: incident.go [hunk 1]
```diff
@@ -176,15 +176,21 @@ func (c *Client) ListIncidentNotes(id string) ([]IncidentNote, error) {
...
-	ID        string    `json:"id,omitempty"`
-	Summary	  string    `json:"summary,omitempty"`
-	CreatedAt string    `json:"created_at,omitempty"`
-	AlertKey  string    `json:"alert_key,omitempty"`
+	APIObject
+	CreatedAt   string       `json:"created_at,omitempty"`
+	Status      string       `json:"status,omitempty"`
+	AlertKey    string       `json:"alert_key,omitempty"`
+	Service     APIObject    `json:"service,omitempty"`
+	Body        interface{}  `json:"body,omitempty"`
```
Comment (soullivaneuh) [inline incident.go]: I don't know how to deal with that not working field.

It has various of data even a key that are not represented on the official API documentation like `cef_details`.

I would like to store it as a "Various mixed array", but I don't know how to do to make json unmarshal happy.

Tried `interface{}` and `[]interface{}` without success:

```
json: cannot unmarshal number into Go value of type []pagerduty.IncidentAlert
```

Any help would be appreciated.

**Comment by soullivaneuh** [comment]
I finally found the issue: I simply forgot to manage the pagination.

**Comment by mattstratton** [comment]
Can you please rebase this so that the CircleCI config gets into your branch so that the Circle tasks will run?

**Comment by soullivaneuh** [comment]
@mattstratton Done.

The only missing thing is a pagination, but I don't really know how to handle it.

Shall I create a new `ListIncidentAlertsOptions`?

**Pull request merged**